flink-cdc-connect/flink-cdc-source-connectors/flink-connector-tidb-cdc/src/main/java/org/apache/flink/cdc/connectors/tidb/TDBSourceOptions.java

```diff
@@ -82,10 +90,13 @@ private TDBSourceOptions() {}
                     .withDescription("TiKV GRPC batch scan concurrency");
 
     public static TiConfiguration getTiConfiguration(
-            final String pdAddrsStr, final Map<String, String> options) {
+            final String pdAddrsStr, final String hostMapping, final Map<String, String> options) {
```

Comment:
Could we add a new method named `getTiConfiguration(
            final String pdAddrsStr, final Map<String, String> options) {
            final String pdAddrsStr, final String hostMapping, final Map<String, String> options)` instead of changing the exits one ?

Reply:
hostMapping is an optional variable and returns tiConfiguration.Adding a new method would increase the complexity of the code. Change to Optional.of(new UriHostMapping(hostMapping)).ifPresent(tiConf::setHostMapping);